Library/Formula/llvm.rb

```diff
@@ -84,6 +99,10 @@ def install
       fail "The Python bindings need the shared library."
```

Comment:
@mikemcquaid This is meaningless to me. Because there's no option to turn off python bindings. Currently we only have the choice to build against system python or brew python.